File: matriz_5x5.py
"""
matriz = [[0 for _ in range(5)] for _ in range(5)]

for i in range(5):
    for j in range(5):
        valor = int(input(f"Ingrese el valor para la fila [{i}] columna [{j}]: "))
        matriz[i][j] = valor

print("\nMATRIZ:")
for fila in matriz:
    for elemento in fila:
        print(elemento, end="\t")
    print()
"""

v = [[0 for _ in range(3)] for _ in range(3)]
print(v)

File: matriz_5x5.java
import java.util.Scanner;

public class Matriz5x5 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int[][] matriz = new int[5][5];

        for (int i = 0; i < 5; i++) {
            for (int j = 0; j < 5; j++) {
                System.out.print("Fila [" + i + "] Columna [" + j + "]: ");
                matriz[i][j] = sc.nextInt();
            }
        }

        System.out.println("\nMATRIZ:");
        for (int i = 0; i < 5; i++) {
            for (int j = 0; j < 5; j++) {
                System.out.print(matriz[i][j] + "\t");
            }
            System.out.println();
        }

        sc.close();
    }
}